inst/scripts/query-pubmed-grants.R

```diff
@@ -1,26 +1,27 @@
-## ----setup, include = FALSE-----------------------------------------------------------------------------------------------------------------------------------------------------------
-install.packages('librarian')
-install.packages("synapser", repos=c("http://ran.synapse.org", "http://cran.fhcrc.org"))
+# ----setup, include = FALSE-----------------------------------------------------------------------------------------------------------------------------------------------------------
+# options(repos = c(CRAN = "https://cloud.r-project.org/"))
+# remotes::install_cran('rentrez')
+# remotes::install_cran('librarian')
+# remotes::install_version('rjson', version='0.2.21')
+# remotes::install_version('reticulate', version='1.28')
+# remotes::install_cran("synapser", repos = c("http://ran.synapse.org", "https://cloud.r-project.org"))
+
+# install.packages("synapser", repos=c("http://ran.synapse.org", "http://cran.fhcrc.org"))
 
 librarian::shelf(
   optparse,
   rmarkdown,
-  reticulate,
   janitor,
   dplyr,
   readr,
   stringr,
-  reticulate,
-  synapser,
   easyPubMed,
   comprehenr,
-  easyPubMed,
   httr,
   tidyr,
```

Comment:
Future improvement: you don't necessarily want to install tidyr, it is a HUGE library, and looking at this script, there are only a handful of packages that could reduce the installation time.